pkg/docker/containers-view.jsx

```diff
@@ -319,7 +364,8 @@ var ImageList = React.createClass({
     getInitialState: function () {
         return {
             images: [],
-            pulling: []
+            pulling: [],
+            vulnerableInfos: {}
```

Comment:
Usually we would separate the data store from the view, so have more data in the props. But I definitely don't think it's worth changing this now. We can have a general discussion about this sometime in the near future and consider whether your approach should become the default.